readme.md

```diff
@@ -95,13 +95,19 @@ The key difference you have to remember is that the rendering result isn't a DOM
 To ensure all examples work and you can begin your adventure with Ink, make sure to set up Babel with a React preset. After [installing Babel](https://babeljs.io/docs/en/usage), configure it in `package.json`:
 
 ```json
-{
-	"babel": {
-		"presets": [
-			"@babel/preset-react"
-		]
-	}
-}
+  "babel": {
+    "presets": [
+      "@babel/preset-react",
+      [
+        "@babel/preset-env",
+        {
+          "targets": {
```

Comment:
Is it possible to skip `targets` to make the config shorter? Also, could you use tabs here, not spaces?

Reply:
if you want something that works, in node, this is the most reliable way to do it.
https://babeljs.io/docs/en/babel-preset-env#targetsnode
happy to add this
https://prettier.io/docs/en/options.html#tabs

Reply:
Nope, no need to add prettier, you can just convert spaces to tabs by hand.